### DISCUSSION TITLE
Azure Cosmos DB no SQL RAG API integration

### DISCUSSION THREAD

**Discussion by Bergdis** [danny-avila/LibreChat#3271] [Feature Requests & Suggestions]
Are there any plans to integrate more langchain vectorstore connectors into the RAG API codebase?
I have a request for this connector: 
https://python.langchain.com/v0.2/docs/integrations/vectorstores/azure_cosmos_db_no_sql/

If not I can look into the implementation, but wanted to see if it is in the roadmap :)

**Comment by Spyros-Vl** [comment]
That would be really useful, no SQL offers a pay as you go plan I think!
